src/main/java/com/redhat/cloud/notifications/routers/InternalService.java

```diff
@@ -11,18 +11,24 @@
 import javax.inject.Inject;
 import javax.validation.Valid;
 import javax.validation.constraints.NotNull;
+import javax.ws.rs.Consumes;
 import javax.ws.rs.DELETE;
 import javax.ws.rs.GET;
 import javax.ws.rs.NotFoundException;
 import javax.ws.rs.POST;
 import javax.ws.rs.PUT;
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
 import javax.ws.rs.core.Response;
 import java.util.List;
 import java.util.UUID;
 
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
 @Path("/internal")
+@Consumes(APPLICATION_JSON)
+@Produces(APPLICATION_JSON)
```

Comment:
I thought this was the default and we didn't need to specify it?

Reply:
That is true for `quarkus-resteasy` but apparently we have to add these annotations back with `quarkus-reasteasy-reactive`. I suspect the reactive extension is missing some bytecode enhancement. I'll investigate that later (or talk with the resteasy-reactive team).